service/k8sclient/k8sclient.go

```diff
@@ -1615,21 +1701,19 @@ func getResources(resources common.ResourceList) (cpuMillis uint64, memoryBytes
 	return cpuMillis, memoryBytes, nil
 }
 
-// GetConsumedCPUAndMemory returns consumed CPU and Memory in given namespace. If namespaces
-// is empty slice, it tries to get them from all namespaces.
-func (c *K8sClient) GetConsumedCPUAndMemory(ctx context.Context, namespaces ...string) (
+// GetConsumedCPUAndMemory returns consumed CPU and Memory in given namespace. If namespace
+// is empty, it tries to get them from all namespaces.
+func (c *K8sClient) GetConsumedCPUAndMemory(ctx context.Context, namespace string) (
```

Comment:
I stumbled on this error while fixing GetPods method. There is no need for multiple namespaces. Even if we passed more namespaces to the old implementation, it would not work because kubectl supports working only with one namespace at a time.